lti13migration.php

```diff
@@ -0,0 +1,27 @@
+<?php
+// This file is part of the EQUELLA module - http://git.io/vUuof
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle. If not, see <http://www.gnu.org/licenses/>.
+ echo "
+  <h2>Update oEQ resource links to use LTI 1.3</h2>
+  <div>
+    <div>
+      Click <a href='?action=download'>here</a> to download the CSV of courses that will be affected by the migration.
```

Comment:
I know this is kind of a one use page, however the use of 'click here' is a throw back to early web and generally considered poor for accessibility. Should we perhaps bump upgrade this page a bit and use buttons?

For example:

```html
<p>
  Before starting the migration, you can download a CSV which will list all courses which will
  be updated. This CSV can be used for review, and no system modifications will occur.
<p>
<form action="?action=download" method="GET">
  <button>Download CSV</button>
</form>
```

I know it's onerous to need a whole form, but the other options around buttons and links are also not really aligned with accessibility.

If this approach is followed, the same should be done for the migration link below.

Reply:
Yeah let's make this page a bit better. I will use `form` .